src/cogs/when_next.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import datetime
+
+import discord
+from discord.ext import commands
+
+from interactive import (
+    InteractionPipeline,
+    ChoiceInteraction,
+)
+
+DATE_FORMAT = "%d/%m/%y"
+FROM_KEY = "--from"
+UNTIL_KEY = "--until"
+
+COG_HELP = f"""
+    Starts the process of finding out when people are next available to hang out
+    Arguments:
+        {FROM_KEY}: Specifies the first day that can be selected to hang out on
+        {UNTIL_KEY}: Specifies the last day that can be selected to hang out on 
+"""
+
+class WhenNext(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.logging = logging.getLogger(__name__)
+
+    @commands.command(name="whennext")
+    async def entry(self, context, cmd: str):
+        self.logging.info("whennext called")
+        arg_dict = self._parse_command(cmd)
+        from_date = datetime.date.today()
+        if FROM_KEY in arg_dict:
+            from_date = self._parse_date(context, arg_dict[FROM_KEY])
+            if from_date is False:
+                return
+        until_date = from_date + datetime.timedelta(weeks=2)
+        if UNTIL_KEY in arg_dict:
+            until_date = self._parse_date(context, arg_dict[UNTIL_KEY])
+            if until_date is False:
+                return
+        weekend_days = []
+        while from_date < until_date:
+            if from_date.weekday() > 4:
+                weekend_days.append(from_date.strftime(DATE_FORMAT))
+            from_date = from_date + datetime.timedelta(days=1)
+        ipl = InteractionPipeline(
+            ChoiceInteraction(*weekend_days)
+        )
+        await ipl.send_and_watch(
+            context.channel,
+            discord.embed(
```

Comment:
************* Module cogs.when_next
src/cogs/when_next.py:52:12: E1101: Module 'discord' has no 'embed' member; maybe 'Embed'? (no-member)

Reply:
All linting issues fixed as of 1324b2b